src/CoreBundle/Resources/config/services.xml

```diff
@@ -5,13 +5,18 @@
 
     <parameters>
         <parameter key="vendor_dir">%kernel.root_dir%/../vendors</parameter>
+        <parameter key="chameleon_system_core.const.path_outbox" type="constant">PATH_OUTBOX</parameter>
```

Comment:
Could we use a bundle config here?

Reply:
not for the constants - since we may or may not want to use them.

The other two parameters could be moved to a bundle config - so i'll do that

Reply:
configuration has been added